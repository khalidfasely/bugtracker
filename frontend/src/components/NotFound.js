import { Link } from "react-router-dom";

const NotFound = () => (
    <div>
        404 - Page Not Found!
        <Link to="/">Go Home.</Link>
    </div>
);

export default NotFound;