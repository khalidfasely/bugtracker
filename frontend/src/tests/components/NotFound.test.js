import React from "react";
import { render } from '@testing-library/react';
import NotFound from '../../components/NotFound';

test('Should render NotFound component correctly', () => {
    const { asFragment } = render(<NotFound />);
    expect(asFragment()).toMatchSnapshot();
});