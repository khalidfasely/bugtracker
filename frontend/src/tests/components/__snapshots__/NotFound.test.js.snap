// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFound component correctly 1`] = `
<DocumentFragment>
  <div>
    404 - Page Not Found!
    <a>
      Go Home.
    </a>
  </div>
</DocumentFragment>
`;
